dotnet-efcore/EntityFrameworkCoreVsDapper/Tests/PersonRepositoryDapperUnitTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using EntityFrameworkCoreVsDapper.Repositories;
+using Assert = Xunit.Assert;
+
+namespace Tests
+{
+    public class PersonRepositoryDapperUnitTests
+    {
+        private PersonsRepositoryDapper repository = new PersonsRepositoryDapper();
+
+        [Fact]
+        public void When_Quering1000Persons_ThenReturn1000Objects()
+        {
+            var queryResults = repository.Get_1000_Persons();
+
+            Assert.True(1000 == queryResults.Count());
```

Comment:
Would be cleaner to use both List.Count property and Assert.Equal(1000, queryReults.Count), as the intention is an equality check.